### PR TITLE
Edit profile pages.

### PR DESCRIPTION
**Pull request by IvanAndrukh**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

Create profile view and edit pages.

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/static/src/containers/profile/Profile.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        };
+    }
+
+    componentWillMount(){
+        this.getProfile(+this.props.match.params.profileId);
+    }
+
+    getProfile = (profileId) => {
+        getProfileService(profileId).then(response => {
+            console.log(response);
```
Comment (lhalam) [inline eventually/static/src/containers/profile/Profile.js]: ??? console.log

File: eventually/static/src/containers/profile/Profile.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                id: response.data['user'],   
+                first_name: response.data['first_name'],
+                middle_name: response.data['middle_name'], 
+                last_name: response.data['last_name'],
+                hobby: response.data['hobby'],
+                photo: response.data['photo'],
+                birthday: new Date(response.data['birthday'])
+            });
+        }, error => {
+            console.log(error);
```
Comment (lhalam) [inline eventually/static/src/containers/profile/Profile.js]: console.log

File: eventually/static/src/containers/profile/Profile.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        const photo = this.state.photo;
+        const birthday = this.state.birthday.getFullYear() + '-' +(this.state.birthday.getMonth() + 1) + '-' +  this.state.birthday.getDate();
+        
+
+        putProfileService(this.state.id, first_name, middle_name, last_name, hobby, photo, birthday)
+            .then(response => {
+                this.handleClose();
+            })
+            .catch(function (error) {
+                console.log(error);
```
Comment (lhalam) [inline eventually/static/src/containers/profile/Profile.js]: console.log

File: eventually/static/src/containers/profile/ProfileEdit.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import FileUpload from '../fileUpload/FileUpload';
 import TextField from 'material-ui/TextField';
 import RaisedButton from 'material-ui/RaisedButton';
 import DatePicker from 'material-ui/DatePicker';
+import {blueGrey800,orange500} from 'material-ui/styles/colors';
```
Comment (lhalam) [inline eventually/static/src/containers/profile/ProfileEdit.js]: space `{blueGrey800, orange500}`

File: eventually/static/src/containers/profile/ProfileView.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {getProfileService} from './ProfileService';
+import ProfileEdit from './ProfileEdit';
+import {getImageUrl} from '../../helper';
+import {getUserId} from '../../helper/utils';
+import Paper from 'material-ui/Paper';
+import RaisedButton from 'material-ui/RaisedButton';
+
```
Comment (lhalam) [inline eventually/static/src/containers/profile/ProfileView.js]: reformat this page

**Review by lhalam — COMMENTED** [review]
please, run ESLint

File: eventually/static/src/containers/profile/ProfileEdit.js (hunk 4 above)
Comment (IvanAndrukh) [reply eventually/static/src/containers/profile/ProfileEdit.js]: I dont need this colors so I deleted them.

File: eventually/static/src/containers/profile/ProfileView.js (hunk 5 above)
Comment (IvanAndrukh) [reply eventually/static/src/containers/profile/ProfileView.js]: Done.

File: eventually/static/src/containers/profile/Profile.js (hunk 1 above)
Comment (IvanAndrukh) [reply eventually/static/src/containers/profile/Profile.js]: done

File: eventually/static/src/containers/profile/Profile.js (hunk 3 above)
Comment (IvanAndrukh) [reply eventually/static/src/containers/profile/Profile.js]: done

**Review by redmyt — APPROVED** [review]

**Review by Silent777 — APPROVED** [review]

**Pull request merged**